dgl_sparse/include/sparse/sparse_matrix.h

```diff
@@ -137,6 +137,26 @@ class SparseMatrix : public torch::CustomClassHolder {
   static c10::intrusive_ptr<SparseMatrix> FromDiag(
       torch::Tensor value, const std::vector<int64_t>& shape);
 
+  /**
+   * @brief Create a SparseMatrix by index slicing.
```

Comment:
Could you ask chatgpt to polish the documentation? Here is what I got from it.
```cpp
/**
 * @brief Create a SparseMatrix by selecting rows or columns based on provided indices.
 *
 * This function allows you to create a new SparseMatrix by selecting specific rows
 * or columns from the original SparseMatrix based on the provided indices. The selection
 * can be performed either row-wise or column-wise, determined by the 'dim' parameter.
 *
 * @param dim Select rows (dim=0) or columns (dim=1).
 * @param ids A tensor containing the indices of the selected rows or columns.
 *
 * @return A new SparseMatrix containing the selected rows or columns.
 *
 * @note The 'dim' parameter should be either 0 (for row-wise selection) or 1 (for column-wise selection).
 * @note The 'ids' tensor should contain valid indices within the range of the original SparseMatrix's dimensions.
 * @note The returned SparseMatrix may share data with the original matrix, depending on the implementation.
 */
c10::intrusive_ptr<SparseMatrix> IndexSelect(int64_t dim, torch::Tensor ids);
```

Reply:
Sure, do you think yours is ok? I can just use it.

Reply:
You still need to carefully check the content from chatgpt.